examples/observability/Makefile

```diff
@@ -0,0 +1,103 @@
+JAEGER_VERSION ?= v1.38.0
+LFC_NAMESPACE ?= keptn-lifecycle-controller-system
+PODTATO_NAMESPACE ?= podtato-kubectl
+
+.PHONY: install
+install:
+	@echo "-----------------------------------"
+	@echo "Create Namespace and install Jaeger"
+	@echo "-----------------------------------"
+	kubectl create namespace observability --dry-run=client -o yaml | kubectl apply -f -
+	kubectl apply -f https://github.com/jaegertracing/jaeger-operator/releases/download/$(JAEGER_VERSION)/jaeger-operator.yaml -n observability
+	kubectl wait --for=condition=available deployment/jaeger-operator -n observability --timeout=120s
```

Comment:
timeout is too short, this step failed on my machine

Reply:
changed this to 300s